File: Dinner/Python/perfect_numbers.py
for number in range (1,1001):
	sum = 0
	count = 1
	while(count <= number):
		if(number % count == 0):
			sum += count
		count+= 1
	perfect = sum - number
	if perfect == number:
		print(f"{number} is a PerfectNumber")

File: Dinner/JavaScript/PerfectNumbers.js
for (let number = 1; number <= 1000; number++){	
	let sum = 0;
	for(let count = 1; count <= number; count++){
		if(number % count == 0) 
		sum = sum + count;

 	}

	let perfect = sum - number ;		
	if(perfect == number)
	console.log(number + " is a perfect number");
					
}

	



